### ISSUE TITLE
Unable to use, there is no problem with the proxy

### ISSUE THREAD

**Issue by bbqspeed** [Solana0x/GrassNode#8]
![1720635927156](https://github.com/Solana0x/GrassNode/assets/119077526/e646376e-dea3-4059-9b1f-7a3b1a5d5b3e)

The proxy line has no issues with other grass projects and can be used normally

**Comment by RanSack620** [comment]
I think there is a problem with proxies.fo.

**Comment by kaicao90** [comment]
Is there another place to rent cheap proxies?

**Comment by bbqspeed** [comment]
> 还有其他地方可以租用便宜的代理吗？

Yes, I am currently using a fixed residential agent IP for 2 RMB per month
